### ISSUE TITLE
Add prev/next links at the bottom of HTML GLs chapters

### ISSUE THREAD

**Issue by hcayless** [TEIC/Stylesheets#267]
See comment in TEIC/TEI#1577

**Comment by martindholmes** [comment]
I think these sorts of changes would be made in P5/Utilities/guidelines.xsl.model, wouldn't they? Rather than the Stylesheets repo. Not that it's worth moving the tickets again.

**Comment by jamescummings** [comment]
@martindholmes Yes, I think that is the right place to put it, but since it isn't about the text of the guidelines but the XSL that produces it, makes sense to have the tickets here so they get seen by the Stylesheet group.

**Comment by ebeshero** [comment]
Council subgroup in Kraków notices that the work on this ticket has already been completed on review of the published Guidelines chapters.
